appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/admin/cli/ListTimers.java

```diff
@@ -146,6 +146,9 @@ public void execute(AdminCommandContext context) {
 
     private String[] listTimers(String[] serverIds) {
         String[] result = new String[serverIds.length];
+        for(int i = 0; i < result.length; i++) {
+            result[i] = "0";
```

Comment:
Is `"0"` special timer? Would empty `""` do?

Reply:
The "0" is purely for correct reflection on the DAS UI, I imagine returning an empty string would then present an empty string for the value of timer count (ie number of timers on an instance) 0 is much more accurate than nothing for an instance with no timers